pkgs/development/python-modules/sievelib/default.nix

```diff
@@ -12,9 +12,15 @@ buildPythonPackage rec {
 
   patches = [
     (fetchpatch {
+      name = "pip-10-pip-req.patch";
       url = "https://github.com/tonioo/sievelib/commit/1deef0e2bf039a0e817ea6f19aaf1947dc9fafbc.patch";
       sha256 = "0vaj73mcij9dism8vfaai82irh8j1b2n8gf9jl1a19d2l26jrflk";
     })
+    (fetchpatch {
+      name = "requirements-in-setup-py.patch";
+      url = "https://github.com/tonioo/sievelib/pull/73.patch";
```

Comment:
this isn't a stable url

Reply:
`https://github.com/tonioo/sievelib/commit/91f40ec226ea288e98379da01672a46dabd89fc9.patch` should be the same while being stable.

Reply:
I suppose if github disappeared, with the latter we at least can track down the right commit from elsewhere, I sometimes veer towards the PR because it gives a lot more context about the patch.

Reply:
if you go to the github commit url, it will point to the containing PR and branches.

My main concern is that if someone were to force push or push to the branch, then the sha will differ and the fetchpatch will fail

Reply:
Can that _happen_ post-merge?